src/payment/strategies/cybersource/cybersource-payment-strategy.ts

```diff
@@ -79,7 +86,7 @@ export default class CyberSourcePaymentStrategy implements PaymentStrategy {
         const paymentData = payment.paymentData as CreditCardInstrument;
 
         return this._cardinalClient.configure(clientToken)
-            .then(() => this._cardinalClient.runBindProcess(paymentData.ccNumber))
+            .then(() => this._cardinalClient.runBindProcess(this._getBinNumber(paymentData)))
```

Comment:
If this can take either a `ccNumber` or a `iin`, could you update the variable name inside `runBindProcess` to read something like `binNumber` instead of `ccNumber`?

In fact, I'm not sure if this method should be called `runBindProcess` or `runBinProcess`... 😕

Reply:
Yeah, you're right. `runBinProcess` is the correct one